pkg/utils/flow/flow.go

```diff
@@ -230,12 +229,9 @@ func (e *execution) runNode(ctx context.Context, id TaskID) {
 	e.stats.Running.Insert(id)
 
 	go func() {
-		start := e.flow.Clock.Now().UTC()
-
 		log.V(1).Info("Started")
 		err := node.fn(ctx)
-		end := e.flow.Clock.Now().UTC()
-		duration := end.Sub(start)
+		duration := e.flow.clock.Now().UTC().Sub(e.flow.clock.Now().UTC())
```

Comment:
Sorry for my wrong suggestion. Now, the duration will always be `0s`. We can't inline the `start` variable.